tests/manage/ocs_288.py

```diff
@@ -0,0 +1,292 @@
+    cmd = "ceph fs ls"
+    log.info(fs_name)
+    out = tools_pod.exec_ceph_cmd(ceph_cmd=cmd)
+    if out:
+        out = out[0]['name']
+        log.info(out)
+        if out == fs_name:
+            log.info("FileSystem got created from Ceph Side")
+            ceph_validate = True
+        else:
+            log.error("FileSystem was not present at Ceph Side")
+            return False
+    result = CFS.get(resource_name=fs_name)
+    if result['metadata']['name']:
+        log.info(f"Filesystem got created from kubernetes Side")
+        k8s_validate = True
+    else:
+        log.error("Filesystem was not create at Kubernetes Side")
+        return False
+    return True if (ceph_validate and k8s_validate) else False
+
+
+def create_multiple_rbd_storageclasses(count=1, ):
+    """
+    Function for creating multiple rbd storageclass
+    :param count:
+    :return:
+    """
+
+    for sc_count in range(count):
+        log.info("Creating CephBlockPool")
+        pool_obj = helpers.create_ceph_block_pool()
+        helpers.create_storage_class(
+            constants.CEPHBLOCKPOOL,
+            interface_name=pool_obj.name,
+            secret_name=RBD_SECRET_OBJ.name
+        )
+
+    return True
+
+
+def get_cephblockpool():
+    sc_obj = POOL.get()
+    sample = sc_obj['items']
+    pool_list = [
+        item.get('metadata')['name'] for item in sample
+    ]
+    return pool_list
+
+
+def get_secret():
+    sc_obj = SECRET.get()
+    sample = sc_obj['items']
+    conside_secret = "csi-"
+    secret_list = [
+        item.get('metadata')['name'] for item in sample if (
+            conside_secret in item.metadata.name
+        )
+    ]
+    return secret_list
+
+
+def get_pvc():
+    sc_obj = PVC.get()
+    sample = sc_obj['items']
+
+    pvc_list = [
+        item.get('metadata')['name'] for item in sample
+    ]
+    return pvc_list
+
+
+def get_storageclass():
+    """
+    Function for getting all storageclass
+    :return:
+    """
+    sc_obj = SC.get()
+    sample = sc_obj['items']
+
+    ignore_sc = 'gp2'
+    storageclass = [
+        item.get('metadata')['name'] for item in sample if (
+            item.get('metadata')['name'] not in ignore_sc
+        )
+    ]
+    return storageclass
+
+
+def create_pvc(storageclass_list, count=1):
+    """
+    Function for creating pvc and multiple pvc
+    :param kwargs:
+    :return:
+    """
+    for i in range(count):
+        sc_name = random.choice(storageclass_list)
+        pvc_obj = helpers.create_pvc(sc_name)
+        log.info(f"{pvc_obj.name} got Created and got Bounded")
+    return True
+
+
+def create_storageclass_cephfs():
+    """
+    Function for creating CephFs storageclass
+    :return:
+    """
+    helpers.create_storage_class(
+        constants.CEPHFILESYSTEM,
+        helpers.get_cephfs_data_pool_name(),
+        CEPHFS_SECRET_OBJ.name
+    )
+
+    return True
+
+
+@tier1
+@pytest.mark.usefixtures(
+    ocs288_fixture.__name__,
+)
+class TestCaseOCS288(ManageTest):
```

Comment:
add docstring with the link to the TC in Polarion

Reply:
I don't think that having internal url in the docstring is a good idea.
Btw do we have a convention for that already? I was thinking about eg. using pytest markers for this, so that one could just be able to ask to run all tests which covers given polarion test case. See eg. https://docs.pytest.org/en/latest/usage.html#record-property (if I recall right, such feature has been implemented by someone from cloud forms qe team to link polarion id in a test case so that it's visible in junit xml report, and usable for importing into polarion).